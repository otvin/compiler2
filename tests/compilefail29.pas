program compilefail29(output);
{cannot define a constant using unknown identifier}

const pi = 3.14;
	negpi = -a;

var a:integer;

begin
	writeln(pi);
	writeln(negpi);
end.
