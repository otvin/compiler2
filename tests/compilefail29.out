Undefined Identifier: 'a' in file:tests/compilefail29.pas line:5     column:14 
